### PR TITLE
swagg: build paths stripped of basePath

### PR DESCRIPTION
**Pull request by fenollp**

### CODE REVIEW

**Comment by fenollp** [comment]
Formatter bug [minified](https://github.com/fenollp/erlang-formatter/pull/3) & [reported](https://bugs.erlang.org/browse/ERL-204).

File: core/kazoo_ast/src/cb_api_endpoints.erl [hunk 1]
```diff
@@ -420,59 +414,42 @@ format_path_token(<<"_", Rest/binary>>) ->
...
-        {'match', [<<"accounts">>]} -> <<?CURRENT_VERSION/binary, "/accounts">>;
-        {'match', [<<"api_auth">>]} -> <<?CURRENT_VERSION/binary, "/api_auth">>;
-        {'match', [<<"basic_auth">>]} -> <<?CURRENT_VERSION/binary, "/basic_auth">>;
-        {'match', [<<"google_auth">>]} -> <<?CURRENT_VERSION/binary, "/google_auth">>;
-        {'match', [<<"ip_auth">>]} -> <<?CURRENT_VERSION/binary, "/ip_auth">>;
-        {'match', [<<"schemas">>]} -> <<?CURRENT_VERSION/binary, "/schemas">>;
-        {'match', [<<"shared_auth">>]} -> <<?CURRENT_VERSION/binary, "/shared_auth">>;
-        {'match', [<<"sup">>]} -> <<?CURRENT_VERSION/binary, "/sup">>;
-        {'match', [<<"system_configs">>]} -> <<?CURRENT_VERSION/binary, "/system_configs">>;
-        {'match', [<<"token_auth">>]} -> <<?CURRENT_VERSION/binary, "/shared_auth">>;
```
Comment (fenollp) [inline core/kazoo_ast/src/cb_api_endpoints.erl]: @jamesaimonetti Was this a typo? Or on purpose?

Comment (jamesaimonetti) [inline core/kazoo_ast/src/cb_api_endpoints.erl]: Typo it appears

Comment (fenollp) [inline core/kazoo_ast/src/cb_api_endpoints.erl]: Ok fixed in 2fbca2a

**Pull request merged**